napari/_vispy/canvas.py

```diff
@@ -77,8 +78,10 @@ def bgcolor(self):
 
     @bgcolor.setter
     def bgcolor(self, value):
-        _value = self._background_color_override or value
-        SceneCanvas.bgcolor.fset(self, _value)
+        color = Color(self._background_color_override or value)
+        # to get correct blending, set canvas alpha to zero
+        color.alpha = 0
+        SceneCanvas.bgcolor.fset(self, color)
```

Comment:
```suggestion
        _value = self._background_color_override or value
        SceneCanvas.bgcolor.fset(self, _value)
```
I think it's safe to revert this. dst_alpha isn't used anymore and this change requires fixing tests plus it messes up rendering on Qt6 (see my comment at bottom of thread).

Reply:
`DST_ALPHA` is used for additive blending though. Have you tried with that?

Reply:
But that's overridden for the bottom layer when it could blend with canvas, so the canvas alpha doesn't feed into that DST-alpha, right?

Reply:
True! Will revert then :)